geometry_msgs/PoseStamped target_pose
string ik_link_name
---
sensor_msgs/JointState joint_state
int16 error_code
